qiskit/synthesis/discrete_basis/solovay_kitaev.py

```diff
@@ -74,13 +79,20 @@ def load_basic_approximations(self, data: list | str | dict) -> list[GateSequenc
 
         # if a file, load the dictionary
         if isinstance(data, str):
-            data = np.load(data, allow_pickle=True)
+            data = np.load(data, allow_pickle=True).item()
 
         sequences = []
-        for gatestring, matrix in data.items():
+        for gatestring, matrix_and_phase in data.items():
+            if isinstance(matrix_and_phase, tuple):
+                matrix, global_phase = matrix_and_phase
+            else:
+                matrix, global_phase = matrix_and_phase, 0
```

Comment:
Is setting the global phase to zero equivalent to the previous code which didn't set the global phase in the sequence?

Reply:
Yes, exactly 🙂